packages/vscode-extension/src/commands/BaseAction.ts

```diff
@@ -41,11 +58,13 @@ export default abstract class BaseAction {
 	}
 
 	protected async runSuiteCloudCommand(args: { [key: string]: string } = {} ) {
+        this.logToOutputExecutionDetails();
+
 		return new SuiteCloudRunner(this.executionPath).run({
 			commandName: this.commandName,
 			arguments: args,
 		});
```

Comment:
We should add a new line after this so in that way the next output doesn't get mixed with this one.

Reply:
Added the newline.